projects/plugins/jetpack/modules/custom-css.php

```diff
@@ -28,7 +28,7 @@ function jetpack_load_custom_css() {
 function custom_css_loaded() {
 	_deprecated_hook(
 		'custom_css_loaded',
-		'jetpack-$$next-version$$',
+		'$$next-version$$',
```

Comment:
This is what is causing the build to fail at the moment.

For this to work, everything needs to be on one line as per the docs here - https://github.com/Automattic/jetpack/blob/trunk/projects/packages/README.md#package-version-annotations:

So this should work: 


```
// translators: %s is a link to the WordPress.org documentation.
_deprecated_hook( 'custom_css_loaded', 'jetpack-$$next-version$$', 'WordPress Custom CSS', sprintf( esc_html__( 'Jetpack no longer supports Custom CSS. Read the WordPress.org documentation to learn how to apply custom styles to your site: %s', 'jetpack' ), 'https://wordpress.org/documentation/article/styles-overview/#applying-custom-css' ) );
```

Possibly adding the translatable string to a variable to then make it more readable.